roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -105,14 +105,15 @@
     name: >-
           {%- if ansible_os_family in ['RedHat', 'Suse'] -%}
           iputils
-          {%- else -%}
+          {%- elif ansible_os_family == "Debian" -%}
```

Comment:
Isn't the change L116 enough ? While would you also need this one ?

Reply:
It's helpful to improve the Robustness.

It can work in a OS(not Redhat,Suse,Debian), but has the `ping` command.

Reply:
I'm just wondering if everything will be fine for other ansible_os_family, like ClearLinux or Flatcar or any others

Reply:
(well if `iputils-ping` is only available on Debian then you're right with your change)

Reply:
Thanks @floryut , 
I check the os support list, and also fix the issue in the Flatcar.

Reply:
Thank you 👍